src/vs/workbench/contrib/terminal/browser/xterm/quickFixAddon.ts

```diff
@@ -344,26 +343,14 @@ export async function getQuickFixesForCommand(
 							}
 							case TerminalQuickFixType.Opener: {
 								const fix = quickFix as ITerminalQuickFixOpenerAction;
-								const label = localize('quickFix.opener', 'Open: {0}', fix.uri.toString());
+								const label = localize('quickFix.opener', 'Open: {0}', fix.uri.scheme + fix.uri.path + fix.uri.authority);
```

Comment:
to do - is there a function that does this?

Reply:
trying joinPath

Reply:
👇

https://github.com/microsoft/vscode/blob/968026a46db6ad769e0d9f3a048d2c7d780de083/src/vs/workbench/services/label/common/labelService.ts#L195

Reply:
there isn't but should be. going to add